rdr_client/work_queue.py

```diff
@@ -2,15 +2,15 @@
     You can only return results for the same awardee that the service account is connected to.
 
     Run this using ./run_client.sh --project all-of-us-rdr-prod --account [your account]
-    --service_account [service account]@all-of-us-rdr-prod.iam.gserviceaccount.com work_queue.py
+    --service_account [service account]@email.com work_queue.py
```

Comment:
You can remove @email.com, too. The service account value is the whole thing, include stuff after the at sign.

Reply:
done